### ISSUE TITLE
IllegalArgumentException: Unknown column requested: _data

### ISSUE THREAD

**Issue by akohout** [jdamcd/android-crop#47]
In Google Play I have the following stack trace

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.krittiq.app/com.soundcloud.android.crop.CropImageActivity}: java.lang.IllegalArgumentException: Unknown column requested: _data
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2184)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2233)
at android.app.ActivityThread.access$800(ActivityThread.java:135)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1196)
at android.os.Handler.dispatchMessage(Handler.java:102)
at android.os.Looper.loop(Looper.java:136)
at android.app.ActivityThread.main(ActivityThread.java:5001)
at java.lang.reflect.Method.invokeNative(Native Method)
at java.lang.reflect.Method.invoke(Method.java:515)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:785)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:601)
at de.robv.android.xposed.XposedBridge.main(XposedBridge.java:132)
at dalvik.system.NativeStart.main(Native Method)
Caused by: java.lang.IllegalArgumentException: Unknown column requested: _data
at android.database.DatabaseUtils.readExceptionFromParcel(DatabaseUtils.java:167)
at android.database.DatabaseUtils.readExceptionFromParcel(DatabaseUtils.java:137)
at android.content.ContentProviderProxy.query(ContentProviderNative.java:413)
at android.content.ContentResolver.query(ContentResolver.java:461)
at android.content.ContentResolver.query(ContentResolver.java:404)
at com.soundcloud.android.crop.CropUtil.getFromMediaUri(CropUtil.java:95)
at com.soundcloud.android.crop.CropImageActivity.setupFromIntent(CropImageActivity.java:126)
at com.soundcloud.android.crop.CropImageActivity.onCreate(CropImageActivity.java:79)
at android.app.Activity.performCreate(Activity.java:5231)
at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1087)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2148)
... 12 more
```

Device: n7000 (Android 4.4)

**Comment by alexblack** [comment]
I *think* that PR might do the job, let me know if its not good and I can see about writing one.

**Comment by alexblack** [comment]
thoughts?

**Comment by alexblack** [comment]
PR submitted: https://github.com/jdamcd/android-crop/pull/70

**Comment by jdamcd** [comment]
Merged #70 to fix the crash. It should also show some progress UI, though, since the copy can be slow.
